bin/makef.sh

```diff
@@ -81,12 +99,12 @@ echo "### reconnected loopback to ${loopback}"
 echo "### creating filesystems"
 mkfs.vfat -n EFI ${loopback}p2
 mkfs.ext4 -L USR -E lazy_itable_init=0,lazy_journal_init=0 ${loopback}p3
-mkfs.ext4 -L ROOT -E lazy_itable_init=0,lazy_journal_init=0 ${loopback}p4
+mkfs.ext4 -L ROOT -O quota -E lazy_itable_init=0,lazy_journal_init=0,quotatype=usrquota:grpquota:prjquota  ${loopback}p4
 # part of debian-cloud-images, I am sure we want that :-) -> it is default
 #tune2fs -c 0 -i 0 ${loopback}p3
 
 echo "### mounting filesystems"
-mkdir -p ${dir_name}          && mount ${loopback}p4 ${dir_name}
```

Comment:
here on mount of the fs i think we do not need this right? it would be needed later in the OS like you specified later ... but not here

Reply:
I wasn't sure. I added the mount option just to verify that even when the image is being build, the project quota would be kept up to date. Seems to be not expensive to do, and might prevent issues later.

Reply:
yeah but is not supported in my next pr :-D

Reply:
Let me check, if it's not required I'll remove it.

Reply:
top

Reply:
If the filesystem is mounted without the `prjquota` flag, then anything added to the filesystem at that time won't be tracked. 

If the filesystem is remounted with the flag, any files added later are correctly tracked.

For this use-case, that behavior is good enough, so I'll remove the flag from the `mount` operation in the script.